ui/components/app/modals/confirm-remove-account/confirm-remove-account.test.js

```diff
@@ -47,15 +75,15 @@ describe('Confirm Remove Account', () => {
     expect(props.hideModal).toHaveBeenCalled();
   });
 
-  it('should call remove account with identity address', async () => {
+  it('should call remove account with account address', async () => {
```

Comment:
Nit: I know this is not from your PR but still
```suggestion
  it('should call method removeAccount with account address', async () => {
```

Reply:
Sure, no problem. Changed in 19731a32e04def4e5b573cb76cea4a26451cf118